### PR TITLE
OD-202: Add information about optional dependency to sensio/generator-bundle

### PR DESCRIPTION
**Pull request by yshyshkin**

### CODE REVIEW

**Comment by orocla** [comment]
Thank you for your pull request.

It looks like this may be your first contribution to an Oro, Inc. open source project, in which case you'll need to sign a Contributor License Agreement (CLA) at http://www.orocrm.com/contributor-license-agreement/

If you've already signed the CLA, it's possible we don't have your GitHub username or you're using a different email address. GitHub uses the email address you set in your local Git configuration to associate commits with your GitHub account. Please sign the CLA again using the correct GitHub username and email address or see this help article on [setting the email on your git commits](https://help.github.com/articles/setting-your-email-in-git/).

Once you've signed the CLA, please allow for some time for the submission to be processed.

File: cookbook/how_to_create_new_bundle.rst [hunk 1]
```diff
@@ -66,6 +66,13 @@ Also new bundle can be generated using `Symfony command generate:bundle`_:
 Important thing is that you shouldn't update Kernel and routing - OroPlatform provides it's own way to do that -
 it will be described in the chapter `Enable bundle`_ and in future articles.
```
Comment (nnenasheva) [inline cookbook/how_to_create_new_bundle.rst]: Edit: It is important that you don't need to update Kernel and routing, as OroPlatform provides its own way to do that, which will be described in the `Enable bundle`_ chapter and in following articles.

File: cookbook/how_to_create_new_bundle.rst [hunk 2]
```diff
@@ -66,6 +66,13 @@ Also new bundle can be generated using `Symfony command generate:bundle`_:
...
 
+.. note::
+
+    Automatic bundle generation is provided by ``sensio/generator-bundle`` package that defined in ``require-dev``
+    section of ``composer.json`` file in Oro Platform repository - so, if you want to use automatic generation, please,
+    make sure that this package was installed (e.g. you can execute ``composer update`` at project root directory
+    to get all packages from ``require-dev`` section).
+
```
Comment (nnenasheva) [inline cookbook/how_to_create_new_bundle.rst]: Edit: Automatic bundle generation is provided by the ``sensio/generator-bundle`` package, which is defined in the ``require-dev``section of the ``composer.json`` file in the OroPlatform repository. Therefore, in order to use automatic generation, please, make sure that this package has been installed (one of the ways to do so is to execute ``composer update`` at the project's root directory to get all packages from the ``require-dev`` section).

**Comment by yshyshkin** [comment]
Fixed.

**Pull request merged**